packages/types/src/clerk.ts

```diff
@@ -730,6 +730,16 @@ export type UserButtonProps = {
   userProfileProps?: Pick<UserProfileProps, 'additionalOAuthScopes'>;
 };
 
+type PrimitiveKeys<T> = {
+  [K in keyof T]: T[K] extends string | boolean | number | null ? K : never;
+}[keyof T];
+
+type PrimitiveOrganizationResource = PrimitiveKeys<OrganizationResource>;
+type PrimitiveUserResource = PrimitiveKeys<UserResource>;
```

Comment:
We probably don't need these intermediate types, let's use `PrimitiveKeys<OrganizationResource>` directly